safari/Config.xcconfig

```diff
@@ -1,2 +1,4 @@
 MARKETING_VERSION = 21.12.12
 CURRENT_PROJECT_VERSION = 16
```

Comment:
I wonder if we should keep these in the repo too.

- `MARKETING_VERSION` can be set by a custom `version` script inside an additional  `GeneratedOverrides.xcconfig` file
- `CURRENT_PROJECT_VERSION` I think Xcode takes care of this? I think I've seen it automatically bump this number before